rpi_backlight/__init__.py

```diff
@@ -27,10 +35,17 @@ class Backlight:
     """Main class to access and control the display backlight power and brightness."""
 
     def __init__(
-        self, backlight_sysfs_path: Union[str, "PathLike[str]"] = _BACKLIGHT_SYSFS_PATH
+        self,
+        backlight_sysfs_path: Union[str, "PathLike[str]"] = None,
```

Comment:
```suggestion
        backlight_sysfs_path: Optional[Union[str, "PathLike[str]"]] = None,
```

(needs import added of course)

```diff
-  from typing import Any, Callable, Generator, Union, TYPE_CHECKING
+  from typing import Any, Callable, Generator, Union, Optional, TYPE_CHECKING
```